lib/orocos/name_service.rb

```diff
@@ -141,6 +141,11 @@ def reachable?
             validate
             true
         rescue
+            Orocos::CORBA.warn "Name service is unreachable."
```

Comment:
This is NameServiceBase, which is used also by name services that have nothing to do with CORBA. The method you are looking for would be `CORBA::NameService#validate`

Reply:
Thanks for pointing me towards the right location once again! I hope I finally managed to put the warning into the right place.